pkg/access/reviewer.go

```diff
@@ -11,7 +11,8 @@ import (
 type Reviewer struct {
 	ReviewerID string `json:"reviewerId" dynamodbav:"reviewerId"`
 	// Request is the associated request.
-	Request Request `json:"request" dynamodbav:"request"`
+	Request        Request `json:"request" dynamodbav:"request"`
+	SlackMessageID string  `json:"slackMessageId" dynamodbav:"slackMessageId"`
```

Comment:
I was thinking some more about this and following our other struct design patterns I think we should make this as below.

It just means that we can expand this in future and know that these are all notifications

``` go

type Notifications struct {
    // if slack is in use, slack message ID should be populated when this has been notified
    SlackMessageID *string `json:"slackMessageId" dynamodbav:"slackMessageId"`
}

type Reviewer struct {
	ReviewerID string `json:"reviewerId" dynamodbav:"reviewerId"`
	// Request is the associated request.
	Request        Request `json:"request" dynamodbav:"request"`
        Notifications Notifications `json:"notifications" dynamodbav:"notifications"`

}

```